deploy/decco/templates/tcp-cert.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
I have a feeling this secret is not used anywhere and we might be able to delete it.

Reply:
I'm always happy to remove extraneous stuff, but I want to make sure I understand why.

AFAICT, the only App that gets deployed to a Space in the decco Namespace is kplane-usermgr, which has one HTTP endpoint. Still, removing this secret would prohibit TCP endpoints from being added in the future. Is that right?

Reply:
Unless things have changed, AFAIK kplane-usermgr is never deployed into the decco namespace. Normally, a space resource is created in the decco namespace, resulting in a new namespace (the "kplane namespace"), where kplane-usermgr is then deployed. Is my understanding correct?

Reply:
Never mind, I think I'm wrong. Space resources do get created in decco ns, but they need the tcp-cert secret. So ignore what I said. It's all well.

Reply:
👍